Library/Formula/fig.rb

```diff
@@ -68,9 +64,24 @@ def install
 
     bin.install Dir[libexec/"bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+    ln_s bin/"docker-compose", bin/"fig"
   end
 
   test do
-    system "#{bin}/fig", "--version"
+    system "#{bin}/docker-compose", "help", "build"
+  end
+
+  def caveats; <<-EOS.undent
+      Docker Compose requires Docker and Boot2Docker to be installed to function.
+      Each of these packages can be installed thru homebrew, or manually installed.
+
+        brew install boot2docker
+        brew install docker
+
+      Or, to manually install follow instructions here:
+
+        http://docs.docker.com/installation/mac/
+
+    EOS
```

Comment:
I think we can remove these caveats; they are already implied/specified by the dependencies above.